### ISSUE TITLE
Ctrl-O doesn't work when configured as kill-line

### ISSUE THREAD

**Issue by amosbird** [gdh1995/vimium-c#959]
**Describe the bug**

I've specified the following configuration to setup `ctrl-o` as line killer:

```
map <c-o:i> editText run="extend,forward,line,exec,delete"
```

However, when pressing `ctrl-o` in a INSERT context, it still behaviors as "Go back in history". I'm sure this hotkey is controlled by vimium-c because when vimium-c is disabled, `ctrl-o` does nothing.

The "Show available commands" also doesn't show any hints about `ctrl-o`.

**Browser, OS and Vimium C versions**

- Browser name: vivaldi
- Browser version: 6.1.3035.75 (Stable channel) stable (64-bit) 
- Vimium C version: 1.99.99
- OS name and version: Archlinux

**Comment by gdh1995** [comment]
Sorry but I can not reproduce it on a fresh Vivaldi 6.1.3035.84 x64 on Win11 with the published Vimium C v1.99.99 (in fact 1.99.992).

So, a mostly potential reason is, your desktop environment of Archlinux intercepted `Ctrl+O` and change it to some other shortcuts. Or you Vivaldi intercepted it.

You may test what keys Vimium C can get in insert mode, by:
1. open Vimium C Options page
2. Find the "option" named "`Test a key`"
3. Enable the option and then its right input box will be "writable"
4. Focus the input box using a mouse, and then type Ctrl+O
5. look at what the input box shows

![image](https://github.com/gdh1995/vimium-c/assets/5547703/c9042119-dcac-4807-8593-675badb74805)

BTW, do you have other key mappings which map `<c-o>` ?

**Comment by amosbird** [comment]
> BTW, do you have other key mappings which map <c-o> ?

Yes. I have `map <c-o:i> editText run="extend,forward,line,exec,delete"`

Test a key doesn't work as pressing <key>c-o</key> results in (empty), so as <key>c-i</key>. I'm sure it's intercepted by vimium-c, because when vimium-c is disabled, ctrl-o does nothing. When it's enabled, ctrl-o does "Go back in history"

**Comment by gdh1995** [comment]
Sorry I can not guess the root cause. Could you provide your whole configuration file of Vimium C? You may export it into a JSON file by clicking the `Export settings` button on Vimium C Options/
